canvasobject.go

```diff
@@ -75,6 +75,14 @@ type Focusable interface {
 	TypedKey(*KeyEvent)
 }
 
+// Preeditable describes any CanvasObject that can respond to IME inputs.
+// Originally, it should be a method of Focusable, but only a few widgets need PreeditChanged,
+// and adding a method to Focusable will have a big impact on others, so defined as a new interface.
```

Comment:
I agree with this sentiment - adding it to Focusable would be a breaking change.
However can it be worded less like a workaround. 
i.e. "Widgets that implement `Focusable` may also want to implement `PreeditChanged` to handle IME input?

Reply:
That comment certainly makes the relationship with Focusable clearer. People who see this code for the first time may not understand the deep relationship between Focusable and Preeditable.